.github/workflows/publish-linux-packages.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       ZITI_MAINTAINER: "OpenZiti Maintainers <developers@openziti.org>"
       ZITI_HOMEPAGE: "https://openziti.io"
       ZITI_VENDOR: "NetFoundry Inc."
```

Comment:
TARGETARCH is a bool here, but a string below (I think). Should this be a an arch string instead of a bool?

Reply:
It's always a string for the intents and purposes of this workflow. L44 tests if the packager for the current matrix iteration is `deb` and sets the string to what's expected by `dpkg` if so, and for `rpm` if not.

Reply:
I wasn't familiar with GH action expression language ternery operations. Makes more sense now :+1: